Code/src/megadesk.cpp

```diff
@@ -576,22 +558,16 @@ void loop()
 #ifdef SERIALCOMMS
       writeSerial(command_load, targetHeight, pushCount);
 #endif
-
-      if (targetHeight == 0)
-      {
-        targetHeight = currentHeight;
-      }
```

Comment:
done in loadMemory() instead now.

Reply:
I was reasonably sure there was no way it would ever happen. I'm open to moving if check from line 586 down to 662 so that targetHeight is guaranteed non-zero close to where it actually matters.